src/pykeen/pipeline/api.py

```diff
@@ -662,6 +669,7 @@ def pipeline(  # noqa: C901
     random_seed: Optional[int] = None,
     use_testing_data: bool = True,
     evaluation_fallback: bool = False,
+    filter_validation_when_testing: bool = True,
```

Comment:
Follow-up to https://github.com/pykeen/pykeen/pull/398#discussion_r620142248: should this be true by default? Or should it only get turned to true when `pipeline()` is called from the HPO pipeline?

Reply:
See https://github.com/pykeen/pykeen/pull/398#discussion_r620152475

Reply:
Resolved in 21e413db